oneanddone/tests/functional/conftest.py

```diff
@@ -71,3 +76,10 @@ def task(base_url, is_local):
     if is_local:
         from oneanddone.tasks.tests import TaskFactory
         return TaskFactory.create()
+
+
+@pytest.fixture(scope='session')
```

Comment:
To be able to both assign the task to a user and have a new user check if it can be assigned to him  in the same test, a logout and new login should happen in the same test right? How can I do that? Alternatively, 'assignee' as such doesn't seem to be an attribute of a task. So how can I assign the task at creation without using the UI?